### ISSUE TITLE
Update local dev setup documentation

### ISSUE THREAD

**Issue by kimberscott** [lookit/lookit-api#672]
**TL;DR**: Decide whether we want to use invoke going forward for setup, and add complete documentation of local development setup

**Narrative**
As a developer, I want the steps to run Lookit locally to be clear. Currently, invoke tasks are provided to help with setup but the use of these is not documented & it can be hard to troubleshoot without digging into tasks.py, which seems harder than just working through documented steps.

**Acceptance Criteria**
- [ ] Decision on using invoke
- [ ] tasks.py removed & any dependency on invoke removed if stopping use
- [ ] Full instructions for setting up for local development are available for at least one OS in lookit-docs
  - [ ] versions of all software included (e.g. postgres)
  - [ ] instructions on installing & running celery & rabbitmq-server (& under what circumstances this is needed)
  - [ ] friendly instruction to contact us for instructions for other operating systems

**Implementation Notes**

[Putting this in lookit-api even though it's largely a documentation issue because of the decision about invoke]

Advantages of invoke:
- when it works it's just a few commands to use
- automatically detect OS and run appropriate commands rather than providing different instructions for different systems
- get people to run consistent commands for *serving* locally, which they do more frequently than setup - this might be worth keeping even if we don't use it for installation

Disadvantages of invoke:
- changes to installation process require changing source code instead of documentation (this can be appropriate or inappropriate depending on why changes are happening..)
- one more dependency outside of formal requirements
- if you run into an error it's more opaque what exactly is going wrong and how to pick up from there
- if using on unsupported OS, have to dig into tasks.py to find out what you're trying to do instead of looking at detailed instructions that you might be able to adapt
